test/martha/forceAccessUrl.test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('ava');
+const sinon = require('sinon');
+const config = require('../../common/config');
+const helpers = require('./_helpers.js')
+
+const { marthaV3Handler: marthaV3 } = require('../../martha/martha_v3');
+const apiAdapter = require('../../common/api_adapter');
+
+const terraAuth = "Bearer XYZ_TerraAuthToken";
```

Comment:
In the original tests, these variables were extracted from [the all-encompassing "resources" file](https://github.com/broadinstitute/martha/blob/dev/test/martha/_martha_v3_resources.js) by way of a destructuring assignment.

Defining variables upfront and then creating a new object (vs. extracting variables from a "god config") feels more straightforward. What arguments are there against this approach?